src/index.js

```diff
@@ -2,10 +2,10 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
-import coffeescript from "coffeescript";
+import { compile } from "coffeescript";
 
-import schema from "./options.json";
 import CoffeeScriptError from "./CoffeeScriptError";
```

Comment:
Weird, ideally dirty should warn on this and ask about extension, dirty it is like preparing for ES modules, so no warnings for `require`, but yes for `import`, I will look today

Anyway good job, it is first dirty usage, so we will fix everything today and will start to migrate in other repos

Reply:
Yes, makes sense.